client-web/script.js

```diff
@@ -0,0 +1,801 @@
+const NexmoClient = window.NexmoClient;
+
+console.log('window NexmoClient: ', NexmoClient);
+
+const BASE_URL = "https://v-app-companion.herokuapp.com";
+// const BASE_URL = "https://abdulajet.ngrok.io"
```

Comment:
```suggestion
const BASE_URL = "VAPP_BASE_URL";
```
So the CLI plugin can replace it